pkg/kubernetes/nulecule.js

```diff
@@ -323,6 +371,48 @@ define([
             return;
         };
 
+        function fetch_answers_file() {
+            var deferred = $.Deferred();
+            deferred.notify(_("Reading answers file ..."));
+            if (!client.config)
+                deferred.reject(new Error( _("Unable to read config.")));
```

Comment:
This means that kubernetes successfully connected without a config file. We should fall back to kubeconfig-less mode in this case.